testing/docker.go

```diff
@@ -201,15 +206,13 @@ func (d *DockerContainer) portMapping(selectFirst bool, cPort int) (containerPor
 			// Skip ahead until we find the port we want
 			continue
 		}
-		for _, binding := range bindings {
-
-			hostPortUint, err := strconv.ParseUint(binding.HostPort, 10, 64)
-			if err != nil {
-				return 0, "", 0, err
-			}
 
-			return uint(port.Int()), binding.HostIP, uint(hostPortUint), nil
+		hostPortUint, err := strconv.ParseUint(bindings[0].HostPort, 10, 64)
```

Comment:
Let's keep the current behavior of searching through the available bindings.
Most tests use `dktest` instead of this so this won't affect any existing db driver tests, but I'd like to preserve past functionality.

There were issues with assuming that the first binding exists and that the binding data format is consistent.

Reply:
Thank you, I see now. Using of only the first binding confused me.
I'll rollback it and add nolint.

But if you wish I can replace it with this one (it might be more explicit):
```go
for port, bindings := range d.ContainerJSON.NetworkSettings.Ports {
	if !selectFirst && port.Int() != cPort {
		// Skip ahead until we find the port we want
		continue
	}
	if len(bindings) == 0 {
		continue
	}
	hostPortUint, err := strconv.ParseUint(bindings[0].HostPort, 10, 64)
	if err != nil {
		return 0, "", 0, err
	}

	return uint(port.Int()), bindings[0].HostIP, uint(hostPortUint), nil
}
```